Alloy.Api/Services/AlloyQueryService.cs

```diff
@@ -101,6 +106,7 @@ private async void Run(object state)
                         }
                     }
                 }
+                _hostedServiceHealthCheck.CompletedRun();
```

Comment:
This health check calls in every time the Run loop completes, and the health check verifies the loop is running on schedule